RELEASE_NOTES.md

```diff
@@ -1,6 +1,15 @@
 Release Notes
 =============
 
+v1.0.0-beta.x.x
```

Comment:
[Question] Did i see somewhere we were doing `beta.1.x` here?

Reply:
Yeah, but that needed fixing. We dont know whether its a major/minor bump in the beta till release depending on what the changeset is. just one `x` implies its only a minor.